shared/actions/chat2/index.js

```diff
@@ -1538,20 +1438,11 @@ function* attachmentLoad(action: Chat2Gen.AttachmentLoadPayload) {
       filename: fileName,
       identifyBehavior: RPCTypes.tlfKeysTLFIdentifyBehavior.chatGui,
       messageID: message.id,
-      preview: isPreview,
     }
   )
-
-  try {
-    const result = yield Saga.call(downloadFileRpc.run)
-    if (EngineRpc.isFinished(result)) {
-      yield Saga.put(Chat2Gen.createAttachmentLoaded({conversationIDKey, isPreview, ordinal, path: fileName}))
-    } else {
-      yield Saga.put(Chat2Gen.createAttachmentLoadedError({conversationIDKey, isPreview, ordinal}))
-    }
-  } catch (err) {
-    logger.warn('attachment failed to load:', err)
-    yield Saga.put(Chat2Gen.createAttachmentLoadedError({conversationIDKey, isPreview, ordinal}))
+  const result = yield Saga.call(downloadFileRpc.run)
```

Comment:
i'm worried about ditching the try block and err catching. maybe we don't handle this nicely w/ the downloads

Reply:
I can put back the loaded error action I suppose. I'm not sure what it did though.